archetype-module-refapp/src/main/resources/archetype-resources/pom.xml

```diff
@@ -29,25 +29,107 @@
 	</modules>
 
 	<dependencyManagement>
-		<dependencies>
-			<!-- Importing versions from refapp distro -->
-            <dependency>
-                <groupId>org.openmrs.distro</groupId>
-                <artifactId>referenceapplication</artifactId>
-                <version>${openmrsRefappVersion}</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
-		</dependencies>
+			<dependencies>
+				<!-- Importing versions from refapp distro -->
+				<dependency>
+					<groupId>org.openmrs.distro</groupId>
+					<artifactId>referenceapplication</artifactId>
+					<version>${openmrsRefappVersion}</version>
+					<type>pom</type>
+					<scope>import</scope>
+				</dependency>
+				<dependency>
+					<groupId>junit</groupId>
+					<artifactId>junit</artifactId>
+					<version>${junitVersion}</version>
+				</dependency>
+				<dependency>
+					<groupId>org.junit.jupiter</groupId>
+					<artifactId>junit-jupiter-api</artifactId>
+					<version>${junitJupiterVersion}</version>
+				</dependency>
+				<dependency>
+					<groupId>org.junit.jupiter</groupId>
+					<artifactId>junit-jupiter-engine</artifactId>
+					<version>${junitJupiterVersion}</version>
+				</dependency>
+				<dependency>
+					<groupId>org.junit.jupiter</groupId>
+					<artifactId>junit-jupiter-params</artifactId>
+					<version>${junitJupiterVersion}</version>
+					<scope>test</scope>
+				</dependency>
+				<dependency>
+					<groupId>org.junit.vintage</groupId>
+					<artifactId>junit-vintage-engine</artifactId>
+					<version>${junitJupiterVersion}</version>
+				</dependency>
+				<dependency>
+					<groupId>org.mockito</groupId>
+					<artifactId>mockito-junit-jupiter</artifactId>
```

Comment:
Are we still using mockito?

Reply:
removing any of powermock or mockito seems to break the tests and my guess is the reason might be https://github.com/openmrs/openmrs-core/blob/2479e9051d2de9ea382027c69bf6dc533fd0e574/api/src/test/java/org/openmrs/test/jupiter/BaseContextSensitiveTest.java#L129